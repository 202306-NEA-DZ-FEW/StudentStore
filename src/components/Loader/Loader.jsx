export default function Loader() {
    return (
        <div className='min-h-[80vh] flex items-center justify-center'>
            test loader
        </div>
    );
}
