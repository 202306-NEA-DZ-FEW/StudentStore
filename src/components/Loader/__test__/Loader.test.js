import renderer from "react-test-renderer";
import Loader from "../Loader";

jest.mock("react-spinners", () => ({
    GridLoader: jest
        .fn()
        .mockReturnValue(<div className='mocked-grid-loader' />),
}));

it("renders correctly", () => {
    const tree = renderer.create(<Loader />).toJSON();
    expect(tree).toMatchSnapshot();
});
