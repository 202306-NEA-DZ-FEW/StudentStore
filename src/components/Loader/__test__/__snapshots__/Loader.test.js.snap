// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="min-h-[80vh] flex items-center justify-center"
>
  <div
    className="mocked-grid-loader"
  />
</div>
`;
